src/CMakeLists.txt

```diff
@@ -5,7 +5,19 @@
 # the ROCm hipBLAS implementation, we use the ROCm
 # hipBLAS library's hipblas.h header.
 include(ExternalProject)
-option(H4I_ROCM_HIPBLAS_TAG "Tag to use from the ROCm hipBLAS repository when obtaining its hipblas.h header" "rocm-5.3.0")
+# make sure that HIP_VERSION_MAJOR, HIP_VERSION_MINOR, and HIP_VERSION_PATCH are defined
+if(NOT DEFINED HIP_VERSION_MAJOR)
```

Comment:
Since we are going ahead with hardcoded version we don't need these HIP_VERISON_MAJOR, MINOR and Patch. Please remove them.

Reply:
These should always be defined. Why do you want to remove this safety check?

Reply:
Because we are not going to use them hence having these checks here does not required.

Reply:
I don't understand the benefit of removing a safety check but since you insist I will remove it.